pkg/cloud/openstack/machineactuator.go

```diff
@@ -436,18 +441,32 @@ func (oc *OpenstackClient) providerconfig(providerConfig clusterv1.ProviderConfi
 	return &config, nil
 }
 
-func (oc *OpenstackClient) getKubeadmToken() (string, error) {
-	tokenParams := kubeadm.TokenCreateParams{
-		Ttl: time.Duration(10) * time.Minute,
+func (oc *OpenstackClient) createBootstrapToken() (string, error) {
+	token, err := tokenutil.GenerateBootstrapToken()
+	if err != nil {
+		return "", err
 	}
-	output, err := oc.kubeadm.TokenCreate(tokenParams)
+
+	expiration := time.Now().UTC().Add(TokenTTL)
+	tokenSecret, err := bootstrap.GenerateTokenSecret(token, expiration)
 	if err != nil {
-		glog.Errorf("unable to create token: %v", err)
+		// If this happens, there is a bug somewhere
```

Comment:
Should this panic and exit then? What happens if we get here and the execution continues? Any horrible side-effect?

Reply:
I thought about this as well, and think to panic is better, because the admin sees the problem in kubernetes (crash loop), and does not need to dig into logs.